packages/lib-classifier/src/store/annotations/DrawingAnnotation.spec.js

```diff
@@ -0,0 +1,28 @@
+import DrawingAnnotation from './DrawingAnnotation'
+
+const drawingAnnotation = {
+  task: 'T0',
+  value: [{ frame: 0, toolIndex: 0, toolType: 'point', x: 100, y: 150 }]
+}
+
+describe('Model > DrawingAnnotation', function () {
+  it('should exist', function () {
+    const drawingAnnotationInstance = DrawingAnnotation.create(drawingAnnotation)
+    expect(drawingAnnotationInstance).to.exist()
+    expect(drawingAnnotationInstance).to.be.an('object')
+  })
+
+  it('should throw an error without frame number', function () {
```

Comment:
this and the 'should throw an error without toolIndex number' tests are failing, and I'm not sure why. I'll revisit soon, but pushed up if something quick jumps out.

Reply:
I think we can have the model default tool and frame to 0 with the optional type. The task area of the drawing task should auto select the first tool and the subject viewer will default to the first frame.